azure-pipelines.yml

```diff
@@ -228,7 +228,13 @@ stages:
     steps:
       - task: DownloadPipelineArtifact@2
         inputs:
-          path: /home/vsts/work/1/a
+          path: '$(Build.ArtifactStagingDirectory)'
+      - script: |
+              zip -r $(Build.ArtifactStagingDirectory)/Windows-VS-16-2019-Win32.zip $(Build.ArtifactStagingDirectory)/Windows-VS-16-2019-Win32
+              rm -r $(Build.ArtifactStagingDirectory)/Windows-VS-16-2019-Win32
+              zip -r $(Build.ArtifactStagingDirectory)/Windows-VS-16-2019-x64.zip $(Build.ArtifactStagingDirectory)/Windows-VS-16-2019-x64
+              rm -r $(Build.ArtifactStagingDirectory)/Windows-VS-16-2019-x64
```

Comment:
If you have `path: '$(Build.ArtifactStagingDirectory)' ` does that means it's the script's current directory? If so you could remove the `$(Build.ArtifactStagingDirectory)/` prefix on the zip/rm commands.

Reply:
That path is from previous task, 'task: DownloadPipelineArtifact@2'. In '$(Build.ArtifactStagingDirectory)' are saved artifacts resulted from '- stage: Builds'.
In order to remove the '$(Build.ArtifactStagingDirectory)/' prefix I can add 'cd $(Build.ArtifactStagingDirectory)' before zip/rm commands.

Reply:
Ok. Up to you then, I'm fine with the two variants.